src/modules/mc_att_control/mc_att_control_params.c

```diff
@@ -571,3 +571,17 @@ PARAM_DEFINE_FLOAT(MC_TPA_RATE_D, 0.0f);
  * @group Multicopter Attitude Control
  */
 PARAM_DEFINE_FLOAT(MC_DTERM_CUTOFF, 0.f);
+
+/**
+ * Multicopter air-mode
+ *
+ * The air-mode enables the mixer to increase or decrease the total thrust of the multirotor
```

Comment:
Maybe the comment should mention that it's only disabling "upwards boost".
I think it's right to do that but the comment says `increase or decrease`.